js/animate.js

```diff
@@ -8,17 +8,28 @@ const randomNumberWithinRange = (min, max) => {
 };
 
 const fetchX = (n) => {
-  const deg = n * (72 * (Math.PI / 180));
+  const deg = n * ((360 / numOfMembers) * (Math.PI / 180));
   return (width / 5) * Math.cos(deg);
 };
 
 const fetchY = (n) => {
-  const deg = n * (72 * (Math.PI / 180));
+  const deg = n * ((360 / numOfMembers) * (Math.PI / 180));
   return 160 * Math.sin(deg);
 };
 
 const memberList = document.querySelectorAll('.member_animation');
 
+const handleIntersection = (entries) => {
+  entries.forEach(({ isIntersecting }) => {
+    if (isIntersecting) addClassName();
+    else removeClassName();
```

Comment:
How about?
```JS
isIntersecting ? addClassName() : removeClassName()
```

Reply:
This was a really good one! Thanks, Swaraj!